src/styles/blocks/header.scss

```diff
@@ -1,42 +1,45 @@
+@import "variables.scss";
+
 .header {
   display: flex;
   align-items: center;
   justify-content: space-between;
   box-sizing: border-box;
-
   height: 60px;
   padding: 0 50px;
-
   box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
 
   &__logo {
     height: 40px;
   }
-}
 
-.nav {
-  &__list {
+  .nav__list {
```

Comment:
pls make use of sass nesting for `nav` selector like so
.nav {
  &__list {
    ...
  }
  &__item {
    ...
    }
  }